Minichallenge3/Controllers/MyNarrativesViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  MyNarrativesViewController.swift
+//  Minichallenge3
+//
+//  Created by Adriel Freire on 10/05/19.
+//  Copyright © 2019 Adriel Freire. All rights reserved.
+//
+
+import UIKit
+
+class MyNarrativesViewController: UIViewController {
+    let customView = MyNarrativeViews()
+    var clickedRow: IndexPath?
+
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        return .lightContent
+    }
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let tableView = customView.tableView
+        tableView.delegate = self
+        tableView.dataSource = self
+
+        let nib = UINib(nibName: "ExpandableTableViewCell", bundle: .main)
+        tableView.register(nib, forCellReuseIdentifier: "cell")
+        tableView.scrollToRow(at: IndexPath(item: 0, section: 0), at: .top, animated: false)
+
+    }
+    override func loadView() {
+        self.view = customView
+    }
+
+}
+extension MyNarrativesViewController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 30
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+
+        if  indexPath == clickedRow {
+            let cell = tableView.dequeueReusableCell(withIdentifier: "cell") as? ExpandableTableViewCell
+
+            return cell!
```

Comment:
Catch forçado, apesar de que há uma certeza quando a célula ja foi clicada, em algum momento essa lógica de clickedRow pode dá um bug, e quebrar o app